### PR TITLE
Get help sequence diagram (test)

### PR DESCRIPTION
**Pull request by luke-slowen**
Testing mermaid sequence diagram is rendered correctly when published

### CODE REVIEW

File: profession/get-help-with-architecture.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+  participant Requestor
+  participant DDaT Business Partner
+  participant Portfolio
+  participant Roadmap
+  participant Emergency Triage
+  participant Profession
+
+  Requestor->DDaT Business Partner:Define the need and raise
+  note over DDaT Business Partner: Is this definitely an \narchitecture need?
```
Comment (MercerT77) [inline profession/get-help-with-architecture.md:24]: Is this a typo....\narchitecture.....?

File: profession/get-help-with-architecture.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  DDaT Business Partner<-->Profession: Seek advice
+  Requestor<--DDaT Business Partner:Help to define the need
+  note over DDaT Business Partner: Does this fit within \nan existing portfolio?
```
Comment (MercerT77) [inline profession/get-help-with-architecture.md:27]: and here.....\nan existing........

File: profession/get-help-with-architecture.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  Portfolio<-Roadmap: Yes
+  note over Portfolio: Identify architecture support
+  Portfolio<-->Emergency Triage: If capacity issues, seek advice
+  Requestor<-Portfolio: Confirm support, agree statement of work
+  DDaT Business Partner<-Roadmap: No: confirm line to take (no, but...)
+  Requestor<-DDaT Business Partner: Unable to meet demand
+
+  group Emergency Triage
+    DDaT Business Partner->Emergency Triage: **No: raise for emergency triage
+    note over Emergency Triage: Establish portfolio\nand agree prioritisation
```
Comment (MercerT77) [inline profession/get-help-with-architecture.md:42]: and here......\nand agree.......

File: profession/get-help-with-architecture.md (hunk 1 above)
Comment (luke-slowen) [reply profession/get-help-with-architecture.md:24]: \n is a line break in the mermaid sequence diagram code, these are required

**Comment by MercerT77** [comment]
all looks good

File: profession/get-help-with-architecture.md (hunk 2 above)
Comment (luke-slowen) [reply profession/get-help-with-architecture.md:27]: As above, \n required

File: profession/get-help-with-architecture.md (hunk 3 above)
Comment (luke-slowen) [reply profession/get-help-with-architecture.md:42]: As above, \n required

**Pull request merged**